Makefile

```diff
@@ -118,8 +118,8 @@ run-dev-image:
 		-v $(CURDIR)/tmp/pprof:/tmp/pprof \
 		signalfx-agent-dev /bin/bash
 
-MARKERS ?= not packaging and not installer and not k8s and not windows_only and not deployment and not perf_test
 .PHONY: run-integration-tests
+run-integration-tests: MARKERS ?= not packaging and not installer and not k8s and not windows_only and not deployment and not perf_test
```

Comment:
What does this syntax mean?

Reply:
Since both the `run-integration-tests` and `run-k8s-tests` targets use `MARKERS`, I mistakenly made `MARKERS` a global env var, which would incorrectly set the default markers for `run-k8s-tests` to `not packaging and not installer and not k8s and not windows_only and not deployment and not perf_test`.  This change just makes `MARKERS` target-specific to avoid the conflict.